src/current/molt/migration-overview.md

```diff
@@ -0,0 +1,136 @@
+1. Replicate ongoing changes. Enable continuous replication to keep CockroachDB in sync with the source.
+1. Verify consistency before cutover. Use MOLT Verify to confirm that the CockroachDB data is consistent with the source.
+1. Finalize target schema: Recreate indexes or constraints on CockroachDB that you previously dropped to facilitate data load.
+1. Cut over to CockroachDB. Redirect application traffic to the CockroachDB cluster.
+
+For a practical example of the preceding steps, refer to [Migrate to CockroachDB]({% link molt/migrate-to-cockroachdb.md %}).
+
+## MOLT tools
+
+[MOLT (Migrate Off Legacy Technology)]({% link releases/molt.md %}) is a set of tools for schema conversion, data load, replication, and validation. Migrations with MOLT are resilient, restartable, and scale to large data sets.
+
+MOLT [Fetch](#fetch) and [Verify](#verify) are CLI-based to maximize control, automation, and visibility during the data load and replication stages.
+
+<table class="comparison-chart">
+  <tr>
+    <th>Tool</th>
+    <th>Usage</th>
+    <th>Tested and supported sources</th>
+    <th>Release status</th>
+  </tr>
+  <tr>
+    <td class="comparison-chart__feature"><a href="#schema-conversion-tool"><b>Schema Conversion Tool</b></a></td>
+    <td>Schema conversion</td>
+    <td>PostgreSQL, MySQL, Oracle, SQL Server</td>
+    <td>GA (Cloud only)</td>
+  </tr>
+  <tr>
+    <td class="comparison-chart__feature"><a href="#fetch"><b>Fetch</b></a></td>
+    <td>Initial data load; optional continuous replication</td>
+    <td>PostgreSQL 11-14, MySQL 5.7-8.0+, CockroachDB</td>
+    <td>GA</td>
+  </tr>
+  <tr>
+    <td class="comparison-chart__feature"><a href="#verify"><b>Verify</b></a></td>
+    <td>Schema and data validation</td>
+    <td>PostgreSQL 12-14, MySQL 5.7-8.0+, CockroachDB</td>
+    <td><a href="{% link {{ site.current_cloud_version }}/cockroachdb-feature-availability.md %}">Preview</a></td>
+  </tr>
+</table>
+
+### Schema Conversion Tool
+
+The [MOLT Schema Conversion Tool]({% link cockroachcloud/migrations-page.md %}) converts a source database schema to a CockroachDB-compatible schema. The tool:
+
+- Identifies [unimplemented features]({% link molt/migration-strategy.md %}#unimplemented-features-and-syntax-incompatibilities)
+- Rewrites unsupported [DDL syntax]({% link {{ site.current_cloud_version }}/sql-statements.md %}#data-definition-statements)
+- Applies CockroachDB [schema best practices]({% link molt/migration-strategy.md %}#schema-design-best-practices)
+
+### Fetch
+
+[MOLT Fetch]({% link molt/molt-fetch.md %}) performs the core data migration to CockroachDB. It supports:
+
+- [Multiple migration modes](#migration-modes) via `IMPORT INTO` or `COPY FROM`
+- Data movement via [cloud storage, local file servers, or direct copy]({% link molt/molt-fetch.md %}#data-path).
+- [Concurrent data export]({% link molt/molt-fetch.md %}#best-practices) from multiple source tables and shards
```

Comment:
Is it worth calling out here that this is limited to certain PK types? Or is that too detailed here?

Reply:
I think it's fine to leave it out here, but I added a callout to the Best Practices section this links out to.